core/src/main/java/org/apache/accumulo/core/client/ConditionalWriterConfig.java

```diff
@@ -175,4 +190,32 @@ public String getClassLoaderContext() {
     return this.classLoaderContext;
   }
 
+  private static <T> T merge(T o1, T o2) {
+    if (o1 != null)
+      return o1;
+    return o2;
+  }
+
+  /**
+   * Merge this ConditionalWriterConfig with another. If config is set in both, preference will be
+   * given to this config.
+   *
+   * @param other
+   *          Another ConditionalWriterConfig
+   * @return Merged ConditionalWriterConfig
+   * @since 2.0.0
+   */
+  public ConditionalWriterConfig merge(ConditionalWriterConfig other) {
+    ConditionalWriterConfig result = new ConditionalWriterConfig();
+    result.timeout = merge(this.timeout, other.timeout);
+    result.maxWriteThreads = merge(this.maxWriteThreads, other.maxWriteThreads);
+    result.auths = merge(this.auths, other.auths);
```

Comment:
Auths is initialized to a non-null value, so not sure merge will do anything.

Reply:
You're right. I can change it to `result.auths = this.auths;`

Reply:
would it be possible to make the auths follow the pattern around the durability field?

Reply:
> would it be possible to make the auths follow the pattern around the durability field?

Do you mean initializing it as null similar to how we did with durability so it can be treated like the rest?

Reply:
> Do you mean initializing it as null similar to how we did with durability so it can be treated like the rest?

Yeah and in the getter return Auths.EMPTY when its null.

Reply:
Included in the most recent commit